src/frontend/views/header.tsx

```diff
@@ -30,7 +39,7 @@ export default function Header(){
           <h1 className="font-bold text-2xl py-2 mr-20">My Friday Night List</h1>
           <div className="flex">
             <div className="flex w-72 mr-4">
-              <input className="outline-none w-72 rounded-lg bg-[#212F45] pl-4" type="text" placeholder="Search..." />
+              <input className="outline-none w-72 rounded-lg bg-[#212F45] pl-4" type="text" placeholder="Search..." value={props.value} onChange={(e) => props.setQuery(e.target.value)} onKeyDown={(e)=> {if (e.key === "Enter") props.search()}}/>
```

Comment:
The button should be disabled if the user hasn't entered a search value

```
disabled={props.value.trim() === ""}
```

Reply:
This is not correct.